### PR TITLE
Add labels for all hardware control test case

### PR DESCRIPTION
**Pull request by hu-weihua**
Add labels for all hardware control test case.

Related commands are ``rflash``, ``rpower``, ``rinv``, ``rvitals``, ``rspconfig``, ``rsetboot``,``rbeacon``, and ``reventlog``.

Also including test cases from ``UT_openbmc``.

Related task xcat2/xcat2-task-management#200

### CODE REVIEW

File: xCAT-test/autotest/testcase/rbeacon/cases0 [hunk 1]
```diff
@@ -34,6 +37,7 @@ start:rbeacon_version
 description: this case is to test rbeacon -v and --version output
 os:Linux
 hcp:openbmc
+label:cn_bmc_ready,hctrl_openbmc
```
Comment (tingtli) [inline xCAT-test/autotest/testcase/rbeacon/cases0]: Not sure if this kind of cases need cn_bmc_ready label

**Comment by hu-weihua** [comment]
-----------rflash------------
```
-------------------------------
The labels of cases:
-------------------------------
rflash_delete_no_active                      os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rflash_option_a_file_not_exist               os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rflash_option_a_with_multiple_values         os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rflash_option_a_with_non_existent_id         os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rflash_option_a_without_specify_noderange    os=Linux,hcp=openbmc,mn_only,hctrl_openbmc,
rflash_option_c_against_node                 os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rflash_option_c_file_not_exist               os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rflash_option_c_with_multiple_values         os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rflash_option_c_without_specify_noderange    os=Linux,hcp=openbmc,mn_only,hctrl_openbmc,
rflash_option_check_with_V_against_node      os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rflash_option_d_with_multiple_values         os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rflash_option_d_with_non_existent_dir        os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rflash_option_d_without_specify_noderange    os=Linux,hcp=openbmc,mn_only,hctrl_openbmc,
rflash_option_delete_with_multiple_values    os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rflash_option_delete_with_non_existent_id    os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rflash_option_l                              os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rflash_option_l_with_value                   os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rflash_option_l_without_specify_noderange    os=Linux,hcp=openbmc,mn_only,hctrl_openbmc,
rflash_option_u_file_not_exist               os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rflash_option_u_with_multiple_values         os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rflash_option_u_without_specify_noderange    os=Linux,hcp=openbmc,mn_only,hctrl_openbmc,
rflash_unsupport_multiple_option_a_c         os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rflash_unsupport_multiple_option_a_d         os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rflash_unsupport_multiple_option_a_l         os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rflash_unsupport_multiple_option_a_u         os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rflash_unsupport_multiple_option_c_d         os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rflash_unsupport_multiple_option_c_l         os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rflash_unsupport_multiple_option_c_u         os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rflash_unsupport_multiple_option_l_d         os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rflash_unsupport_multiple_option_l_u         os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rflash_unsupport_multiple_option_u_d         os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rflash_usage                                 os=Linux,hcp=openbmc,mn_only,hctrl_openbmc,
rflash_without_option                        os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
```

**Comment by hu-weihua** [comment]
-----------rpower------------
```
-------------------------------
The labels of cases:
-------------------------------
rpower_bmcreboot_perl_python_output    hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rpower_boot                            cn_bmc_ready,hctrl_general,
rpower_err_noderange                   mn_only,hctrl_general,
rpower_errorcommand_OpenpowerBmc       hcp=openbmc,cn_bmc_ready,hctrl_openpower_ipmi,hctrl_openbmc,
rpower_noderange                       cn_bmc_ready,hctrl_general,
rpower_noderange_nodeps                cn_bmc_ready,hctrl_general,
rpower_off                             cn_bmc_ready,hctrl_general,
rpower_on                              cn_bmc_ready,hctrl_general,
rpower_onstandby                       cn_bmc_ready,hctrl_fsp,arch=ppc64,
rpower_reset                           cn_bmc_ready,hctrl_general,
rpower_softoff                         cn_bmc_ready,hctrl_fsp,hctrl_openbmc,
rpower_stat                            cn_bmc_ready,hctrl_general,
rpower_state                           cn_bmc_ready,hctrl_general,
rpower_status                          cn_bmc_ready,hctrl_general,
rpower_suspend_OpenpowerBmc            hcp=openbmc,cn_bmc_ready,hctrl_openpower_ipmi,hctrl_openbmc,
rpower_wake_OpenpowerBmc               hcp=openbmc,cn_bmc_ready,hctrl_openpower_ipmi,hctrl_openbmc,
rpower_wrongpasswd                     cn_bmc_ready,hctrl_openpower_ipmi,
```

**Comment by hu-weihua** [comment]
-----------rinv------------
```
-------------------------------
The labels of cases:
-------------------------------
rinv_all               cn_bmc_ready,hctrl_general,
rinv_bus               cn_bmc_ready,hctrl_fsp,
rinv_config            cn_bmc_ready,hctrl_fsp,
rinv_cpu               cn_bmc_ready,hctrl_openbmc,
rinv_dimm              hcp=ipmi,arch=x86_64,cn_bmc_ready,hctrl_bmc_ipmi,
rinv_errorcommand      cn_bmc_ready,hctrl_general,
rinv_firm              cn_bmc_ready,hctrl_general,
rinv_guid              hcp=ipmi,arch=x86_64,cn_bmc_ready,hctrl_bmc_ipmi,
rinv_h                 mn_only,hctrl_general,
rinv_help              mn_only,hctrl_general,
rinv_mixnode           cn_bmc_ready,hctrl_general,
rinv_model             cn_bmc_ready,hctrl_general,
rinv_mprom             hcp=ipmi,arch=x86_64,cn_bmc_ready,hctrl_bmc_ipmi,
rinv_noderange_err     mn_only,hctrl_general,
rinv_serial            cn_bmc_ready,hctrl_general,
rinv_uuid              cn_bmc_ready,hctrl_openpower_ipmi,
rinv_v                 mn_only,hctrl_general,
rinv_vpd               cn_bmc_ready,hctrl_openpower_ipmi,
rinv_wrongbmcpasswd    cn_bmc_ready,hctrl_openpower_ipmi,hctrl_fsp,hctrl_openbmc,
```

**Comment by hu-weihua** [comment]
-----------rvitals------------
```
-------------------------------
The labels of cases:
-------------------------------
rvitals_all                        cn_bmc_ready,hctrl_general,
rvitals_altitude                   cn_bmc_ready,hctrl_openbmc,
rvitals_cputemp                    arch=ppc64le,hcp=ipmi,cn_bmc_ready,hctrl_openpower_ipmi,
rvitals_disktemp                   arch=ppc64le,hcp=ipmi,cn_bmc_ready,hctrl_openpower_ipmi,
rvitals_errorcommand               cn_bmc_ready,hctrl_general,
rvitals_fanspeed                   cn_bmc_ready,hctrl_bmc_ipmi,hctrl_openpower_ipmi,hctrl_openbmc,
rvitals_h                          mn_only,hctrl_general,
rvitals_lcds                       hcp=hmc,hcp=ivm,hcp=fsp,hcp=ipmi,cn_bmc_ready,hctrl_fsp,
rvitals_leds                       cn_bmc_ready,hctrl_bmc_ipmi,hctrl_openpower_ipmi,hctrl_openbmc,
rvitals_noderange_err              mn_only,hctrl_general,
rvitals_outputs_format_checking    cn_bmc_ready,hctrl_general,
rvitals_power                      cn_bmc_ready,hctrl_bmc_ipmi,hctrl_openpower_ipmi,hctrl_openbmc,
rvitals_state                      cn_bmc_ready,
rvitals_temp                       cn_bmc_ready,hctrl_general,
rvitals_v                          mn_only,hctrl_general,
rvitals_voltage                    cn_bmc_ready,hctrl_general,
rvitals_wattage                    cn_bmc_ready,hctrl_bmc_ipmi,hctrl_openpower_ipmi,hctrl_openbmc,
```

**Comment by hu-weihua** [comment]
-----------rspconfig------------
```
-------------------------------
The labels of cases:
-------------------------------
rspconfig_autopower                                       hcp=fsp,cn_bmc_ready,hctrl_fsp,
rspconfig_autoreboot                                      os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_bootmode                                        os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_date                                            hcp=fsp,cn_bmc_ready,hctrl_fsp,
rspconfig_decfg                                           hcp=fsp,cn_bmc_ready,hctrl_fsp,
rspconfig_dump_clear                                      os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_dump_download                                   os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_dump_generate                                   os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_dump_list                                       os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_dump_no_option                                  os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_gard                                            os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_gateway_invalid                                 os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_get_hostname                                    os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_hostname_with_error_input                       os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_invalid                                         os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_iocap                                           hcp=fsp,cn_bmc_ready,hctrl_fsp,
rspconfig_ip_invalid                                      os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_ip_null                                         os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_ipsrc                                           os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_list_all                                        cn_bmc_ready,hctrl_bmc_ipmi,hctrl_openbmc,
rspconfig_list_gateway                                    hcp=openbmc,cn_bmc_ready,hctrl_bmc_ipmi,hctrl_openbmc,
rspconfig_list_ip                                         cn_bmc_ready,hctrl_bmc_ipmi,hctrl_openbmc,
rspconfig_list_netmask                                    hcp=openbmc,cn_bmc_ready,hctrl_bmc_ipmi,hctrl_openbmc,
rspconfig_list_ntpservers                                 cn_bmc_ready,hctrl_openbmc,
rspconfig_list_vlan                                       cn_bmc_ready,hctrl_bmc_ipmi,hctrl_openbmc,
rspconfig_netmask_invalid                                 cn_bmc_ready,hctrl_openbmc,
rspconfig_node_invalid                                    os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_noderange_invalid                               os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_ntpserver                                       os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_powerrestorepolicy                              os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_powersupplyredundancy                           os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_set_admin_passwd_with_error_input               os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_set_admin_passwd_with_error_origin_password     os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_set_all                                         cn_bmc_ready,hctrl_openbmc,
rspconfig_set_all_invalid                                 os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_set_gateway                                     os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_set_hostname                                    os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_set_hostname_equal_star_with_bmc_is_hostname    os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_set_hostname_equal_star_with_bmc_is_ip          os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_set_ip                                          os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_set_netmask                                     os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_set_ntpservers                                  cn_bmc_ready,hctrl_openbmc,
rspconfig_set_vlan                                        cn_bmc_ready,hctrl_openbmc,
rspconfig_set_vlan_invalid                                os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_sshcfg                                          hcp=hmc,cn_bmc_ready,hctrl_hmc,hctrl_openbmc,
rspconfig_sshcfg_with_error_input                         os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rspconfig_time                                            hcp=fsp,cn_bmc_ready,hctrl_fsp,
rspconfig_timesyncmethod                                  os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
```

**Comment by hu-weihua** [comment]
-----------rsetboot------------
```
-------------------------------
The labels of cases:
-------------------------------
rsetboot_cd_statcheck          cn_bmc_ready,hctrl_openbmc,hctrl_bmc_ipmi,
rsetboot_default_statcheck     cn_bmc_ready,hctrl_openbmc,hctrl_bmc_ipmi,
rsetboot_group_net             cn_bmc_ready,hctrl_openbmc,hctrl_bmc_ipmi,
rsetboot_h                     mn_only,hctrl_general,
rsetboot_hd_statcheck          cn_bmc_ready,hctrl_openbmc,
rsetboot_help                  mn_only,hctrl_general,
rsetboot_net_statcheck         cn_bmc_ready,hctrl_openbmc,hctrl_bmc_ipmi,
rsetboot_node_invalidaction    cn_bmc_ready,hctrl_openbmc,hctrl_bmc_ipmi,
rsetboot_node_invalidnode      mn_only,hctrl_general,
rsetboot_noderange_net         cn_bmc_ready,hctrl_openbmc,hctrl_bmc_ipmi,
rsetboot_v                     mn_only,hctrl_general,
```

**Comment by hu-weihua** [comment]
-----------rbeacon------------
```
-------------------------------
The labels of cases:
-------------------------------
rbeacon_false      os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rbeacon_help       os=Linux,hcp=openbmc,mn_only,hctrl_openbmc,
rbeacon_null       os=Linux,hcp=openbmc,mn_only,hctrl_openbmc,
rbeacon_stat       os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
rbeacon_version    os=Linux,hcp=openbmc,mn_only,hctrl_openbmc,
```

**Comment by hu-weihua** [comment]
-----------reventlog------------
```
-------------------------------
The labels of cases:
-------------------------------
reventlog_all             cn_bmc_ready,hctrl_general,
reventlog_clear           cn_bmc_ready,hctrl_general,
reventlog_null            mn_only,hctrl_general,
reventlog_numofentries    cn_bmc_ready,hctrl_general,
reventlog_s_openbmc       os=Linux,hcp=openbmc,cn_bmc_ready,hctrl_openbmc,
```

**Pull request merged**